css/css-anchor-position/anchor-center-visibility-change-ref.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<style>
+.container {
+  width: 100px;
+  height: 100px;
+  border: solid 3px;
+  position: relative;
+  margin: 50px;
+}
+
+.anchor {
+  anchor-name: --anchor;
+  position: relative;
+  width: 50px;
+  height: 50px;
+  left: 40px;
+  top: 5px;
+  background: lime;
+}
+
+.target {
+  position-anchor: --anchor;
+  position: fixed;
+  background: cyan;
+  justify-self: anchor-center;
```

Comment:
Is it possible to rewrite the reference to not use anchor positioning features? Reftests should not use the technology being tested.

Reply:
Yes - that's a good point. Done.